src/App/Application.cpp

```diff
@@ -3474,7 +3474,7 @@ std::string Application::FindHomePath(const char* sCall)
 #include <cstdlib>
 #include <sys/param.h>
 
-std::string Application::FindHomePath(const char* call)
+std::string Application::FindHomePath(const char* sCall)

```

Comment:
Using prefixes like this was common practice back in the late 90s due to bad tooling so you couldn't easily inspect types or other attributes. I don't think this archaic naming scheme is the way to go for us.

In this context I assume it was changed to avoid an avalanche of other naming changes in the files to keep consistency in variables.

For this reason I think the change is ok 👍

Reply:
Agreed on all counts :).